lightning/src/ln/channelmanager.rs

```diff
@@ -10367,14 +10375,23 @@ where
 				};
 
 				match result {
-					Ok(()) => ResponseInstruction::NoResponse,
-					Err(e) => match responder {
-						Some(responder) => responder.respond(OffersMessage::InvoiceError(e)),
+					Err(err) => match responder {
+						Some(responder) => {
+							abandon_if_payment(context);
+							responder.respond(OffersMessage::InvoiceError(err))
+						}
```

Comment:
nit: here and below, end match arms with a comma.

Reply:
Added proper commas in [528dc64](https://github.com/lightningdevkit/rust-lightning/commit/528dc644e5b995c10f96fe36fb803bc6433580d7)
Thanks for the pointer!